src/styles/sidebar/components/thread.scss

```diff
@@ -50,6 +50,11 @@
       }
     }
   }
+  &__hidden-toggle-button {
+    // This makes the vertical alignment with thread-collapse chevrons
+    // more precise
+    margin-top: -0.25em;
```

Comment:
If there are multiple such buttons in a chain of replies then I note that this doesn't preserve the spacing between buttons (ie. if you made this margin-top a bit larger, the buttons would overlap).